src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends FigureBase {
+    private int firstSide;
+    private int secondSide;
+
+    public RightTriangle(String color, int firstSide, int secondSide) {
+        super(color);
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability. Instead of passing `firstSide` and `secondSide` directly, you can create local variables inside the method that calls the constructor and then pass these variables to the constructor.